### ISSUE TITLE
Flash messages don't work in production

### ISSUE THREAD

**Issue by gedankenstuecke** [openSNP/snpr#399]
That's another weird bug: 

The flash messages that should pop up to signal success/warnings/failures don't work in production. A simple way to see this is trying to go to `https://opensnp.org/signup` if you're already logged in. There should be a flash-notice/warning that this isn't possible for logged-in users with the redirect to their Dashboard. 

If you do the same thing in your dev-environment it works (plus I even wrote some tests for that and those pass as well, see #397)

**Comment by gedankenstuecke** [comment]
@philippbayer could you have a look into this? I think it's the same issue that we had with the other CSS things but I can't seem to figure it out despite lots of playing around o_O

**Comment by philippbayer** [comment]
I bumped into this too. It's a very strange error. Sometimes they work but then the About and Search popups don't work. Weird.

**Comment by philippbayer** [comment]
Actually, after logging in with Firefox 53.0.3 and Chrome 59.0.3071.86 I *do* see the 'You must be logged out to access this page' flash message on https://opensnp.org/signup 
I also see the 'Successfully signed in' flash on production.

Which browser and version are you on?

**Comment by gedankenstuecke** [comment]
I noticed it on Chrome, `Version 58.0.3029.110 (64-bit)`. In Safari `Version 10.1.1 (11603.2.5)` it works. Let me try updating my Chrome :joy:

**Comment by philippbayer** [comment]
It's weird that such a recent Chrome version wouldn't support this, it's definitely something we'll have to fix.

**Comment by gedankenstuecke** [comment]
It's getting weirder and weirder. 

* I updated Chrome and it still doesn't work. At least not in the regular user account. When open an incognito window, log in there and try it works. 
* I disabled all Extensions in my regular Chrome user: doesn't work nevertheless :woman_shrugging: 
* In a Chrome V57 on Linux it works, with my regular user account. :woman_shrugging: 

tl;dr: my Chrome on MacOS seems to be the issue in this case, not openSNP.

**Comment by philippbayer** [comment]
Well maybe it's a Mac issue, cannot check that :)

**Comment by gedankenstuecke** [comment]
Not really, after all it works in the incognito tab on MacOS. So it's really strange 😱

**Comment by philippbayer** [comment]
Is it a cache thing? Maybe it uses the old CSS from the cache and gets confused

**Comment by gedankenstuecke** [comment]
I tried hard-reloads but maybe that's the issue. Will try a clean
re-install of Chrome at some point.
